src/components/RelationshipPicker.tsx

```diff
@@ -1,43 +1,194 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TextField from '@material-ui/core/TextField';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
 import { makeStyles } from '@material-ui/core/styles';
+// import IconButton from '@material-ui/core/IconButton';
+// import CloseIcon from '@material-ui/icons/Close';
 
 export default function RelationshipPicker(): JSX.Element {
   const classes = useStyles();
 
+  const [state, setState] = useState({
```

Comment:
You should make the same changes here in this file, too.

Reply:
Thanks I updated the hooks and PR, and was going over Relationshop Comp.
Each relationship needs to have a max of 2 text fields and a close option on each text field. 
How should I approach this? hmm